### DISCUSSION TITLE
How do I report an issue?

### DISCUSSION THREAD

**Discussion by jackaround** [weather-gov/api#697] [General]

**Comment by sullynole** [comment]
nco.ops@noaa.gov

https://www.weather.gov/documentation/services-web-api

**Comment by jackaround** [comment]


**Reply by sullynole** [reply]
If only looking for observations, the upstream source is MADIS.  It has an interface to query observations as well: https://madis-data.ncep.noaa.gov/public/sfcdumpguest.shtml
